### PR TITLE
Graphical re-design of pagination

### PR DESCRIPTION
**Pull request by phtan**
This pull request addresses issue #1134 . It is relevant to issue #1180 

I've left out the [Angular Material icon](https://design.google.com/icons/#ic_more_horiz) for the ellipsis. 

The screenshots below show the pagination when the user is on the following pages:

- random page,
- first page, and
- last page.

screenshot from 2016-06-15 03-13-47
screenshot from 2016-06-15 03-13-59
screenshot from 2016-06-15 03-14-15

### CODE REVIEW

**Comment by trang** [comment]
Thanks for the pull request @phtan. Here's my feedback:

1. I forgot to update the issue for the colors: you'll have to change the background of selected page from #66AA00 to #4caf50, since #4caf50 is going to be our primary color.
2. Remove the grey background on hover over the arrows. There's no effects when hovering over the arrows. If you'd like to add an effect, it would be better to change the color rather than the background color.
3. There is a big space between `{lastpage}` and `>`, while there is almost no space between `<` and `1`. The alignment of the `>` arrow seems off as well (it's below the baseline). The spacing and alignment should be equal on both sides.

**Comment by phtan** [comment]
Addressing @trang 's comment above, the pagination now looks like the following:

screenshot from 2016-06-16 00-37-34

File: app/views/helpers/pagination.php [hunk 1]
```diff
@@ -93,27 +93,25 @@ public function display($extramParams = array())
             // the largest numbers (for example, when we're near the
             // end of the list of English sentences).
             'modulus' => 6,
-            'class' => 'pageNumber'
+            'class' => 'pageNumber',
+            'first' => 1,
+            'last' => 1,
+            'ellipsis' => 'more_horiz'
```
Comment (trang) [inline app/views/helpers/pagination.php]: The `ellipsis` property can be removed. It doesn't have any effect.

File: app/webroot/css/layouts/elements.css [hunk 2]
```diff
@@ -736,42 +736,56 @@
...
-    color: white;
+.numbers a:hover {
+    background: #E0E0E0;
+    color: #757575;
 }
 
 .paging .disabled {
     display: inline;
     padding: 3px 5px;
     margin: 3px;
```
Comment (trang) [inline app/webroot/css/layouts/elements.css]: The `display`, `padding` and `margin` can be deleted.
* `display: inline` doesn't have any effect
* the padding and the margin are causing misalignment with the disabled arrow icons

File: app/webroot/css/layouts/elements.css [hunk 3]
```diff
@@ -736,42 +736,56 @@
...
+a.prev {
+    background: #FFFFFF;
 }
 
+a.next {
+   background: #FFFFFF;
+}
+
+md-icon.disabled.material-icons {
+   background: #FFFFFF;
```
Comment (trang) [inline app/webroot/css/layouts/elements.css]: Should be `background: none`. We want a transparent background for the icons.

**Comment by trang** [comment]
Almost there :) I've posted a few more comments on the code.

**Comment by phtan** [comment]
Made changes as per [above comment](https://github.com/Tatoeba/tatoeba2/pull/1195#issuecomment-226298669).

**Pull request merged**